composer.json

```diff
@@ -9,7 +9,7 @@
     "composer/installers": "^2.0",
     "brainmaestro/composer-git-hooks": "^2.8",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
-    "yoast/phpunit-polyfills": "^1.0",
+    "yoast/phpunit-polyfills": "^1.1",
```

Comment:
Are this and the related changes to `composer.lock` intended changes?

Reply:
Yes, otherwise phpunit won't run

Reply:
Thanks! Just wanted to make sure because I've sometimes pushed unintended composer updates by accident.